City-of-Helsinki/drupal-hdbt
# UHF-3922 user login styling

# TASKNAME [UHF-3922](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3922)

Styling and changing the order of user login page

## What was done
* Changed the order of Drupal and tunnistamo logins
* Added template for login form and login page

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme and platformconfig
    * run `make shell`
    * `composer require drupal/hdbt:dev-UHF-3922-user-login-styling`
    * `composer require drupal/helfi_platform_config:dev-UHF-3922-user-login-styling`
    * `drush cr`

## How to test
* Don't login to Drupal and go to [your own instance here]/user/login
* Check that the styles match the [design](https://app.abstract.com/share/d7ebb0ec-2cbe-4809-a6b3-83fd9faeab25) and that they work both in desktop and mobile

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/300
